src/runtime/server.ts

```diff
@@ -14,17 +14,14 @@ import {
   sandboxBinary,
   ensureBinary,
 } from "../utils";
+// @ts-ignore
```

Comment:
this is because `node-port-check` doesn't include types?

Reply:
Yes, I could include types, but I put inside a function.  I could move this to the utils if that makes this part look cleaner.